README.md

```diff
@@ -1,8 +1,59 @@
-# vscode-aks-tools
+# Azure Kubernetes Service (AKS) Extension for Visual Studio Code (Preview)
 
-* View your AKS clusters in the Kubernetes extension cloud explorer
-* Add AKS clusters to kubeconfig via the GUI
-* Run AppLens detector diagnostics on your AKS clusters
+The Visual Studio Code extension for Azure Kubernetes Service helps you view your AKS cluster, merge and save kubeconfig, perform AKS Diagnostics checks, and run AKS Periscope within VS Code.
+
+## Prerequisites
+
+Ensure that you have installed the [Kubernetes Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools) extension for VS Code prior to installing the Azure Kubernetes Service Extension for Visual Studio Code following these [instructions](https://code.visualstudio.com/docs/azure/kubernetes#_install-the-kubernetes-extension).
+
+## Installation
+
+1. Download and install the [Azure Kubernetes Service extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) for Visual Studio Code.
+
+2. Wait for the extension to finish installing then reload Visual Studio Code when prompted.
+
+3. Once the installation is complete, you'll see a section named **Azure** under **Clouds**.
+
+4. Sign in to your Azure Account by clicking **Sign in to Azure…**
+
+## Features
+
+### Merge and Save Kubeconfig
+
+Log-in with your Azure credentials and view all AKS clusters in your Azure subscription(s) within VS Code. Right click on your AKS cluster and click on **Merge into Kubeconfig** to add the selected AKS cluster to the kubeconfig file or click on **Save Kubeconfig** to save the kubeconfig of the selected AKS cluster.
```

Comment:
💡 Note: `Log-in with your Azure credentials and view all AKS clusters in your Azure subscription(s) within VS Code.` -- Is `True` for all the feature for this extension, i.e. I have a feeling Login should be a sub-topic which kind-of explains how with screenshots and then we can talk about all the features available for this extension.

Reply:
I can move the log-in part out since we have it in the instructions already. Instead, I'll add a paragraph in under Features stating something along the lines of, "Once you have successfully logged in with your Azure credentials, you call right click on your AKS cluster to perform following actions."